Source/Core/DolphinQt2/GCMemcardManager.cpp

```diff
@@ -299,10 +299,9 @@ void GCMemcardManager::ExportFiles(bool prompt)
     }
   }
 
-  QMessageBox::information(this, tr("Success"),
-                           tr("Successfully exported %1 %2")
-                               .arg(count)
-                               .arg(count == 1 ? tr("save file") : tr("save files")));
+  QString text = count == 1 ? tr("Successfully exported the save file.") :
```

Comment:
We should probably format these to use the plural forms described in Qt's [Writing Source Code for Translation](https://doc.qt.io/qt-5/i18n-source-translation.html) (wx's equivalent would probably by wxPLURAL) at some point

Reply:
Yes, we should look into that at some point. I just wonder how we should handle making "localized" English strings for plural strings...